### ISSUE TITLE
`git_status()` picker leads to blank buffer

### ISSUE THREAD

**Issue by mroavi** [nvim-telescope/telescope.nvim#1053]
### Description
A blank buffer is opened with Telescope's `git_status()` function when selecting a file that has been modified:

![Peek 2021-07-27 12-55_telescope_git_status_issue](https://user-images.githubusercontent.com/19517248/127142816-c2df17e4-d62b-4bad-96a4-14d9aa070355.gif)

This is how I configure `git_status()` so that the command affects the repo containing the current buffer:

```lua
function M.git_status()
  require("telescope.builtin").git_status{
    cwd = vim.fn.expand("%:p:h"),
  }
end

utils.remap("n", "<Leader>gs", "<Cmd>lua require('mrv.plugins.telescope').git_status()<CR>")

```

**Expected Behavior**
To have the modified file open properly.

**Actual Behavior**
A blank buffer is displayed instead of the modified file.

<details><summary>Environment</summary>

- nvim --version output: 
```
NVIM v0.5.0
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/gcc-11 -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/runner/work/neovim/neovim/build/config -I/home/runner/work/neovim/neovim/src -I/home/runner/work/neovim/neovim/.deps/usr/include -I/usr/include -I/home/runner/work/neovim/neovim/build/src/nvim/auto -I/home/runner/work/neovim/neovim/build/include
Compiled by runner@fv-az242-526
```

- Operating system:
```
NAME="Manjaro Linux"
ID=manjaro
ID_LIKE=arch
BUILD_ID=rolling
PRETTY_NAME="Manjaro Linux"
ANSI_COLOR="32;1;24;144;200"
HOME_URL="https://manjaro.org/"
DOCUMENTATION_URL="https://wiki.manjaro.org/"
SUPPORT_URL="https://manjaro.org/"
BUG_REPORT_URL="https://bugs.manjaro.org/"
LOGO=manjarolinux
```

- Telescope commit: 69adcbc

</details>

**Comment by l-kershaw** [comment]
I can't reproduce this on my machine.

As an attempt to diagnose this problem, could you see if the output of `:echo expand('%:p')` gives the full filepath as you would expect for one of these blank buffers? I noticed in your screen recording that the filepath in your statusline was considerably shorter for the blank buffer.

**Comment by mroavi** [comment]
Hi @l-kershaw. Thanks for looking at this.

Running `:echo expand('%:p')` results in `nvim/.config/nvim/after/ftplugin/julia.vim` which indeed is missing the first part (`~/dotfiles`)

**Comment by l-kershaw** [comment]
I have a few ideas about what might be causing this, and think it might have a similar root cause to #1025, which I am trying to fix in #1026. I'll have more of a look at it later today.

**Comment by Conni2461** [comment]
#1026 was merged closing
